products/jbrowse-web/src/tests/BasicLinearGenomeView.test.js

```diff
@@ -90,6 +90,10 @@ describe('valid file tests', () => {
     fireEvent.click(getSeqMenuItem)
     expect(state.session.views[0].leftOffset).toBeTruthy()
     expect(state.session.views[0].rightOffset).toBeTruthy()
+    const closeButton = await findByTestId('close-seqDialog')
```

Comment:
any reason this extra test condition was added to this PR?

Reply:
I added it because I realize I was only testing opening the dialog. No other reason.

Reply:
kk i was just curious if it was related to non-isolated tests causing issues or something

Reply:
I was checking the coverage report and decided to add it. Not really needed.